perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -343,12 +369,21 @@ sub _send_aggregated_alert_email(%) {
     );
 
     unless ($result) {
-        $token->insert();
+        return { success => 1 };
     } else {
-        print "Failed to send alert $data{alert_id}!";
+        return { error => "failed to send email" };
     }
 }
 
+sub _send_aggregated_alert_phone {
+    my %data = @_;
+    my $result = FixMyStreet::SMS->new(cobrand => $data{cobrand})->send(
+        to => $data{alert_user}->phone,
+        body => sprintf(_("Your report has had an update; please visit %s to view. %s to stop."), $data{problem_url}, $data{unsubscribe_url}),
```

Comment:
I found this a bit hard to understand. I had to read it twice to ensure I was going to click the right link:

![image](https://user-images.githubusercontent.com/4776/98678593-b40f6700-2356-11eb-8190-5ae5e4a940bb.png)

Putting the action before the link, and using a bit of vertical whitespace might help:
```suggestion
        body => sprintf(_("Your report has had an update; to view: %s\n\nTo stop: %s"), $data{problem_url}, $data{unsubscribe_url}),
```

Reply:
(also, could we include the report reference in the message? A user who makes lots of reports could potentially receive multiple identical texts in a short timespan)